leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogMigrator.java

```diff
@@ -98,6 +110,14 @@ public static <V extends Persistable & Versionable> long migrateAndReturnCutoff(
         return context.migrationState().getCutoff();
     }
 
+    private static <V extends Persistable & Versionable> boolean isLeaderNamespace(MigrationContext<V> context) {
+        String paxosLeaderUseCase = "leaderPaxos!learner";
+        String paxosLeaderNamespace = "leaderPaxos";
+
+        return context.namespaceAndUseCase().namespace().value().equals(paxosLeaderNamespace)
+                && context.namespaceAndUseCase().useCase().equals(paxosLeaderUseCase);
```

Comment:
yeah these are the ones we saw. I'd be surprised that `!acceptor` wasn't also flagged (and pretty sure that's also OK), but fine to be strict for now